Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ViewportBehavior/ViewportBehaviorPage.xaml.cs

```diff
@@ -67,28 +65,13 @@ private void ClearLogsButton_Click(object sender, RoutedEventArgs e)
             _logs.Clear();
         }
 
-        private async void EffectElementHost_EnteredViewport(object sender, EventArgs e)
-        {
-            AddLog("Entered viewport");
```

Comment:
This log thing shows up in the example, reason this was removed?

Reply:
It it was not doing anything other than logging, and the compiler was complaining about the function being `async` with no `await`. I can just bring it back for now and remove the `async`. Then when the animation gets reintroduced it would be simpler.